apps/editor.planx.uk/src/components/EditorNavMenu/EditorNavMenu.tsx

```diff
@@ -220,44 +257,103 @@ function EditorNavMenu() {
     return accessibleByCurrentUserRole;
   };
 
-  const visibleRoutes = routes.filter(isRouteAccessible);
+  // Filter accessible routes within each section
+  const visibleSections = sections
+    .map((section) => ({
+      ...section,
+      routes: section.routes.filter(isRouteAccessible),
+    }))
+    .filter((section) => section.routes.length > 0);
+
+  // Count total visible routes across all sections
+  const totalVisibleRoutes = visibleSections.reduce(
+    (count, section) => count + section.routes.length,
+    0,
+  );
 
   // Hide menu if the user does not have a selection of items
-  if (visibleRoutes.length < 2) return null;
+  if (totalVisibleRoutes < 2) return null;
```

Comment:
Is this logic actually still applicable?

Currently, isn't the case that - 
 - Only platform admin will see anything on the main menu (global settings, admin panel, select a team)
 - There will always be >2 routes on the team menu

Maybe we can simplify this with a check which is essentially "don't show the root menu if user is not platformAdmin or analyst"?